### ISSUE TITLE
[FVT] xcattest script doesn't support global variable with "http://"

### ISSUE THREAD

**Issue by caomengmeng** [xcat2/xcat-core#1095]
In default.conf definite a global variable with "http://", then failed to read the value, but only https
default.conf
```
[System]
OS=rhels
ISO=/RHEL-7.2-20151030.0-Server-ppc64-dvd1.iso
STATELITE=rhels7.2-ppc64-statelite-compute
STATELESS=rhels7.2-ppc64-netboot-compute
GATEWAY=10.2.4.17
DATE=2016
MIGRATION1=http:///xcat.org/files/xcat/xcat-dep/2.x_Linux/
```

```
Not to run:
******************************
Start to run test cases
******************************
------START:test_http::Time:Thu May 12 08:27:28 2016------

RUN:echo http

[echo http] Running Time:0 sec
RETURN: rc = 0
OUTPUT:
http

RUN:echo 2016

[echo 2016] Running Time:0 sec
RETURN: rc = 0
OUTPUT:
2016
------END::test_http::Passed::Time:Thu May 12 08:27:28 2016 ::Duration::0 sec------

```

**Comment by tingtli** [comment]
@caomengmeng i think we need to point out which script does not support.  The script should be xcattest, not xCATreg.  It will cause confusion when you mention framework.

**Comment by hu-weihua** [comment]
@caomengmeng, this isn't problem of ``Automation test framework``, this is a problem of ``xcattest`` script.  ``xcattest`` only support limited character for the value of ``System->Varible`` previously, not include ``:`` just as you used.

**Comment by hu-weihua** [comment]
@caomengmeng , I have fixed this issue by pull request #1117. Could you please verify it?

**Comment by caomengmeng** [comment]
Works well now, so close this issue.
